test/icalendar/util/deserialize_test.exs

```diff
@@ -74,6 +74,7 @@ defmodule ICalendar.Util.DeserializeTest do
     assert %Event{} = event
   end
 
+  @tag :skip
```

Comment:
Why is this skipped?

Reply:
This is to show that it is a breaking change. It no longer deserializes to a string. It deserializes to a map. This is the original test, I added a new test for the new functionality.

I left this in (but skipped so we have passing tests) so you can compare the old functionality with the new.

Reply:
Let's update this test to pass now

Reply:
> Let's update this test to pass now

I duplicated in the test below with the new functionality. 

https://github.com/lpil/icalendar/blob/902df35488ded9310d0f3fc0611b60bab57d8735/test/icalendar/util/deserialize_test.exs#L107-L134

So we can just delete this one.

Reply:
@lpil removed